stubs/pyserial/serial/threaded/__init__.pyi

```diff
@@ -32,17 +33,19 @@ class LineReader(Packetizer):
     def handle_line(self, line: str) -> None: ...
     def write_line(self, text: str) -> None: ...
 
-class ReaderThread(threading.Thread):
+_T = TypeVar("_T", bound=Protocol)
```

Comment:
Not a full review yet, but you should add a default to make this backwards compatible:

```suggestion
_T = TypeVar("_T", bound=Protocol, default=Protocol)
```

Also maybe rename to `_P` (`_T` is often used for unbound type vars) and move to the top of the file.